tensorflow_lite_support/python/task/processor/proto/class_pb2.py

```diff
@@ -13,6 +13,63 @@
 # limitations under the License.
 """Class protobuf."""
 
+import dataclasses
+from typing import Any
+
+from tensorflow.tools.docs import doc_controls
 from tensorflow_lite_support.cc.task.processor.proto import class_pb2
 
-Category = class_pb2.Class
+
+@dataclasses.dataclass
+class Category:
+  """Category is a util class, contains a label, its display name, a float
+  value as score, and the index of the label in the corresponding label file.
+  Typically it's used as result of classification tasks.
+
+  Attributes:
+    index: The index of the label in the corresponding label file.
+    score: The probability score of this label category.
+    display_name: The display name of the label, which may be translated for
+      different locales. For example, a label, "apple", may be translated into
+      Spanish for display purpose, so that the `display_name` is "manzana".
+    class_name: The label of this category object.
+  """
+
+  index: int
+  score: float
+  display_name: str
+  class_name: str
```

Comment:
As we rename Class to Category in the Python API, please change `class_name` to `category_name`. I don't think it'll break any tests but please fix if it causes any breakage.

Reply:
@khanhlvg Renamed the attribute and updated all the necessary tests that try to access it.